src/status_im2/contexts/chat/composer/effects.cljs

```diff
@@ -106,12 +106,13 @@
 (defn use-edit
   [{:keys [input-ref]}
    {:keys [text-value saved-cursor-position]}
-   {:keys [edit]}]
+   {:keys [edit]}
+   message-list-rendered?]
   (rn/use-effect
    (fn []
      (let [edit-text        (get-in edit [:content :text])
            text-value-count (count @text-value)]
-       (when (and edit @input-ref)
+       (when (and edit @input-ref @message-list-rendered?)
```

Comment:
I think this needs a more self-explanatory name? smth like `composer-layout-done?`?

Reply:
Done